authors/apps/articles/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from .utils import generate_slug, unique_code_generator
+from django.db.models.signals import pre_save
+from authors.apps.profiles.models import Profile
+from django.contrib.postgres.fields import ArrayField
+
+
+class Tag(models.Model):
```

Comment:
@mariamiah  u can as well as remove this

Reply:
@MuhweziDeo  i have removed it